### DISCUSSION TITLE
How are folks dealing with Sourcemap errors on Firefox?

### DISCUSSION THREAD

**Discussion by oyeanuj** [vercel/next.js#11890] [Help]
Hi folks! As I port one of our apps to NextJS, I'm noticing source-map errors when trying to load the page on Firefox (or Firefox Developers Edition). It works great on Chrome!

```
Source map error: Error: Invalid URL: webpack://[name]_[chunkhash]/webpack/bootstrap
Resource URL: http://lvh.me:1337/_next/static/development/dll/dll_dc8ac110ce5d8530b237.js?ts=1586890384221
Source Map URL: dll_dc8ac110ce5d8530b237.js.map
```

As I looked around, I saw one [an unresolved bug report](https://github.com/zeit/next.js/issues/9471) and an [unanswered issue on StackOverflow](https://stackoverflow.com/q/61011841/4813012).

Since it seems to be a production-stopper for us, I'm curious if there are workarounds here and/or if anyone else experienced this?

**Comment by RidgewellServices** [comment]
Did you find a resolution to this? I am having a similar problem.

**Comment by Vadorequest** [comment]
I don't seem to experience this running https://nrn-v2-mst-aptd-gcms-lcz-sty-c1.now.sh/en under FF `76.0.1`

Source code: https://github.com/UnlyEd/next-right-now/tree/v2-mst-aptd-gcms-lcz-sty

**Comment by Vadorequest** [comment]
@joeytepp @RidgewellServices Check out https://github.com/vercel/next.js/pull/13018, and maybe mention your experience there?
